package.json

```diff
@@ -111,11 +116,11 @@
     "js:allBreaches": "minify public/js/all-breaches/all-breaches.js -o public/dist/all-breaches.min.js",
     "css:minify": "cleancss -o public/dist/app.min.css public/dist/app.min.css",
     "css:concat": "concat -o public/dist/app.min.css public/css/*.css",
```

Comment:
We may want to remove these functions?

Reply:
If we don't need them anymore yeah - err on the side of removing things. :)

Reply:
My bad – we're using the `/dist/app.min.css` on production. I added the step back in on the build step. I removed the `css:concat` function (as the gulp compile does this very same task). Note that `concat` is still being used by JS, so I did not remove the dependency.